### PR TITLE
updated readme2

### PR DESCRIPTION
**Pull request by Auhan99**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
-﻿# DeepChem
+# DeepChem
```
Comment (kyscg) [inline README.md]: Again, I don't see a change here

File: README.md [hunk 2]
```diff
@@ -101,34 +101,34 @@ via this installation procedure.
 ## FAQ and Troubleshooting
 
 1. ```deepchem``` currently supports both Python 2.7 and Python 3.5, and is supported on 64 bit Linux and Mac OSX. Note that DeepChem is not currently maintained for Python 3.6 or with other operating systems. 
-2. Question: I'm seeing some failures in my test suite having to do with MKL
+2. Question: I see some failures in my test suite having to do with MKL
```
Comment (kyscg) [inline README.md]: The original is more appropriate keeping in mind it's present continuous tense nature

**Comment by kyscg** [comment]
Preferably close this and start a fresh PR with only relevant changes

File: README.md (hunk 1 above)
Comment (Auhan99) [reply README.md]: no

**Pull request closed without merging**